frameworks/elastic/src/main/dist/svc.yml

```diff
@@ -6,7 +6,8 @@ pods:
     count: 3
     {{#ENABLE_VIRTUAL_NETWORK}}
     networks:
-      dcos:
+      {{VIRTUAL_NETWORK_NAME_MASTER}}:
```

Comment:
My only concern would be requiring the user to set a bunch of network names rather than just one.

But, this is the most configurable option so /shrug.